scripts/update/Updater.php

```diff
@@ -57,9 +57,9 @@ public function update($initialVersion)
             $this->setVersion('2.6.1');
         }
 
-        $this->skip('2.6.1', '4.3.0');
```

Comment:
I think v4.3.1 failed to be forward ported into develop, that is why. Now the question is does the client needs 4.3.1 ?